From 662d41977012c4cac32c87a48a9919aa805af909 Mon Sep 17 00:00:00 2001
From: Di Zhang <dizhangg@chromium.org>
Date: Thu, 10 Aug 2023 22:00:01 +0000
Subject: [PATCH] [M116 branch 5845] Fix form_controls_not_vertical.css

(cherry picked from commit 2008793de864a026ec5e449e3ef43d7aef3f8270)

Change-Id: I2bd314ea85ef11fe649b62b87e4eb73d7c3a5755
Bug: 1471518
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4765535
Commit-Queue: Di Zhang <dizhangg@chromium.org>
Reviewed-by: Joey Arhar <jarhar@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1181624}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4769891
Reviewed-by: Krishna Govind <govind@chromium.org>
Commit-Queue: Krishna Govind <govind@chromium.org>
Owners-Override: Krishna Govind <govind@chromium.org>
Cr-Commit-Position: refs/branch-heads/5845@{#1355}
Cr-Branched-From: 5a5dff63a4a4c63b9b18589819bebb2566c85443-refs/heads/main@{#1160321}
---

--- a/third_party/blink/renderer/core/css/form_controls_not_vertical.css
+++ b/third_party/blink/renderer/core/css/form_controls_not_vertical.css
@@ -11,6 +11,6 @@
 
 /* TODO(crbug.com/681917): These form controls don't go vertical if
    FormControlsVerticalWritingModeSupport is disabled. */
-input[type="range"], input[type="color"],  input[type="button"], input[type="submit"], input[type="reset"], input[type="file"], textarea, meter, progress, button:not(::-internal-selectmenu-button) {
+input[type="range"], input[type="color"],  input[type="button"], input[type="submit"], input[type="reset"], input[type="file"], textarea, meter, progress, button {
   writing-mode: horizontal-tb !important;
 }
\ No newline at end of file
